x-pack/plugins/apm/public/components/app/mobile/service_overview/metrics_map/embedded_map.tsx

```diff
@@ -125,6 +125,30 @@ function EmbeddedMapComponent({
     }
   }, [embeddable, embeddableRoot]);
 
+  useEffect(() => {
+    const setLayerList = async () => {
+      if (embeddable) {
+        let layerList;
+        switch (selectedMap) {
+          case MapTypes.HTTP:
+            layerList = await getHttpRequestsLayerList(maps);
+
+            break;
+          case MapTypes.SESSIONS:
+            layerList = await getSessionMapLayerList(maps);
+            break;
+          default:
+            layerList = await getHttpRequestsLayerList(maps);
+        }
```

Comment:
you could move the `switch` to another function and make `layerList` a `const`.

Reply:
fixed it: https://github.com/elastic/kibana/commit/e7dd9513cfb75b98918c226c351ae395c31f1de3#diff-5a059c6e17027ba3b6e14d636fbe794bf17357e3164f8c70c435e38b5ea28d24R1